api/src/main/java/org/itsallcode/whiterabbit/api/model/ProjectReportActivity.java

```diff
@@ -10,16 +10,22 @@
 public interface ProjectReportActivity
 {
     /**
+     * Get the {@link Project} of this activity.
+     * 
      * @return the {@link Project} of this activity.
      */
     Project getProject();
 
     /**
+     * Get the total duration of this activity.
```

Comment:
Documentation seems a little bit off given the method name. But I guess it is correct.